bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "purescript-decimals",
+  "version": "1.0.0",
```

Comment:
:+1: 

We probably also want a `repository` field in here so it can be published to Pursuit.